src/resource.rc

```diff
@@ -0,0 +1,3 @@
+aaaa ICON "wx/msw/std.ico"
```

Comment:
This seems very specific to wxWidgets and codeblocks.

Can you isolate this from the other builds?  If it is accidentally included in a build on machines without wxWidgets installed there will be no `wx/msw` folders and the result will be mysterious build errors.

If it absolutely has to be in the src folder, can you at least add some comment describing the purpose of this file and what to do if it causes problems?